index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html ng-app="app">
+    <head>
+        <title>Inverted Index</title>
+        <link rel="stylesheet" href="css/index.css">
+        <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+        <link rel="stylesheet" href="https://code.getmdl.io/1.2.1/material.indigo-pink.min.css">
+        <link rel="stylesheet" href="http://fonts.googleapis.com/css?family=Roboto:300,400,500,700" type="text/css">
```

Comment:
Change the protocol to https so that the request is not blocked when the entire page is loaded over https e.g. on heroku.

You can actually remove the protocol part in all of the and it will default to the protocol used by the page e.g. `<link rel="stylesheet" href="//fonts.googleapis.com/css?family=Roboto:300,400,500,700" type="text/css">`